demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -291,8 +292,9 @@ def insert_script_to_yml(
         imports_to_names = IntegrationScriptUnifier.check_api_module_imports(
             script_code
         )
+        content_path = get_content_path(package_path)
         script_code = IntegrationScriptUnifier.insert_module_code(
-            script_code, imports_to_names
+            script_code, imports_to_names, content_path
```

Comment:
if we don't use `content_path` later, consider just calling `get_content_path` inside the function call